src/rebuild/scan.c

```diff
@@ -721,7 +722,6 @@ rebuild_tgt_scan_handler(crt_rpc_t *rpc)
 	}
 
 	dss_rpc_reply(rpc, DAOS_REBUILD_DROP_SCAN);
-	d_rank_list_free(fail_list);
```

Comment:
I think it is correct to remove the free here, as it is not the correct place.
But a problem is now no place to free the fail_list, i think should register a co_post_reply callback and free it there, but can do in a following patch. The current code seems will cause mem leak.

Reply:
hmm, fail_list is in the reply rpc (rso_ranks_list), I thought it will be freed when rpc is freed?